webpack.config.js

```diff
@@ -1,79 +1,114 @@
-require('shelljs/make');
+'use strict';
+
+var shell = require('shelljs');
 var path = require('path');
-var ExtractTextPlugin = require('sgmf-scripts')['extract-text-webpack-plugin'];
-var jsFiles = require('sgmf-scripts').createJsPath();
-var scssFiles = require('sgmf-scripts').createScssPath();
+var MiniCssExtractPlugin = require('mini-css-extract-plugin');
+var cwd = process.cwd();
+var { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+var packageName = '';
+var jsFiles = {};
+var scssFiles = {};
+process.argv.forEach((val, index) => {
+    if (val === 'entryPackageName') {
+        packageName = process.argv[index + 1];
+        var cssFilesTmp = shell.ls(path.join(cwd, `./cartridges/${packageName}/cartridge/client/**/scss/**/*.scss`));
+        cssFilesTmp.forEach(filePath => {
+            var name = path.basename(filePath, '.scss');
+            if (name.indexOf('_') !== 0) {
+                let location = path.relative(path.join(cwd, `./cartridges/${packageName}/cartridge/client`), filePath);
+                location = location.substr(0, location.length - 5).replace('scss', 'css');
+                scssFiles[location] = filePath;
+            }
+        });
+
+        var jsFilesTmp = shell.ls(path.join(cwd, `./cartridges/${packageName}/cartridge/client/**/js/*.js`));
```

Comment:
Does this contains the file that in `cartridge/client/js/folder_name/.js` ? 
For example `bolt-demandware-managed/cartridges/int_bolt_sfra/cartridge/client/default/js/cart/cart.js`

Reply:
@serenayanbolt In SFCC, it only compile the entry js files which locate in the cartridge/client/js/ directory rather than its sub-directory, and the js included by the entry file would be compiled as well.
So if you want to compile any js file in the sub-directory, please include it in the entry file.

eg.

```
$(document).ready(function () {
    processInclude(require('./cart/cartInstorePickup'));
});
```